### ISSUE TITLE
large distance errors in BBC

### ISSUE THREAD

**Issue by djones1040** [RickKessler/SNANA#93]
Hi Rick - see slack channel discussion.  Even though BBC is finding intrinsic dispersion below 0.1 mag and there are a couple hundred SNe, the distance modulus errors are ~0.3 mag.  To run:

`SALT2mu_fit.pl /project2/rkessler/SURVEYS/WFIRST/ROOT/SALT3/examples/sim_lowzps1_pipeline/pipeline/BBC.input_099`

and the log file is here:

`/project2/rkessler/SURVEYS/WFIRST/ROOT/SALT3/examples/sim_lowzps1_pipeline/Run_GetMu_099/AllSurveys_LOWZ_099/SALT2mu_FITOPT000_MUOPT000.LOG`

Thanks!

**Comment by RickKessler** [comment]
Nothing obvious, but sigma_int -> 0 and reduced chi2 is about 3, so something is way off here. Does the simulated data include an intrinsic scatter model ? Can you try older SNANA versions (ls -d $PRODUCTS/SNANA/SNANA_v10_7*) to see if there's a recent bug ?

**Comment by djones1040** [comment]
yeah I'm trying out GENPERFECT: 1 simulations, so there's no scatter and intrinsic dispersion = 0 is expected.

**Comment by djones1040** [comment]
just an update, BBC distances are still really high-uncertainty with larger biascor

**Comment by djones1040** [comment]
I set up a debug job in `$SNANA_DEBUG/djones/SALT3_BBC`:
```
SALT2mu.exe BBC.input_099
```

**Comment by RickKessler** [comment]
After fixing another problem with input, noticed that MINOS errors are zero; updated SALT2mu so that if MINOS errors are zero, use parabolic error instead.
